index.js

```diff
@@ -10,8 +10,13 @@
 var isUTF8 = require('is-utf8');
 var isBuffer = require('is-buffer');
 
+function maybeUTF8(buf) {
+  // Only "maybe" because we aren't sniffing the whole buffer
+  return isUTF8(buf.slice(0, 7));
```

Comment:
I haven't looked into the internals of `is-utf8`. If it can figure that out with just 7 bytes, then I think this is fine.